BUILDING.md

```diff
@@ -43,10 +43,20 @@ $ make test
 
 To build the documentation:
 
+This will build Node.js first (if necessary) and then use it to build the docs:
+
 ```text
 $ make doc
 ```
 
+If you have an existing Node.js you can build just the docs with:
+
+```text
+$ NODE=node make doc-only
```

Comment:
Thanks for the feedback. I'm going to suggest that the current version is more ideal because I expect the 80% case (really more like 99.9%) when doing `make doc-only` is that the user wants to use the `node` in their path. I did include this note right underneath that:

> (Where `node` is the path to your executable.)

That could be expanded if it's not clear enough:

> (Where `node` can be a pathname: `NODE=/path/to/node make doc-only`)

I suggested the possibility of `npm run doc` for the 80% case and if that happened I could see documenting it like that:

```text
# Use the `node` in the path
npm run doc

# Point to the binary you want to use
NODE=/path/to/node make doc-only
```